### DISCUSSION TITLE
Cannot assign pull requests to people in Android app

### DISCUSSION THREAD

**Discussion by guoxiao** [community/community#53754] [Mobile] [answered]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [X] Projects

### Body

I want to assign a pull request in the Android app, but it says "Issues are disabled" and failed to assign. The assign features works well in the web so I think it's a bug?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by fanglong-it** [comment]
If you are certain that the app should be using issues and you believe that the error is a bug, you can try the following steps:

Check the app's settings to ensure that issues are enabled and that the app is set up to use the same issue tracking system as the web version.

Try logging out of the app and logging back in again. This can sometimes help to clear up issues with permissions or configuration.

If the issue persists, you may need to report the bug to the app's developers or support team. Provide as much detail as possible, including screenshots and a detailed description of the steps you took before encountering the error.

It is also possible that there is an issue with the pull request itself, such as a merge conflict or other problem that is preventing it from being assigned. In this case, you may need to troubleshoot the pull request before you can assign it.

**Reply by guoxiao** [reply]
The issue is that the app is wrongly checking the status of the `Issues` feature. The issue feature is disabled for the repo, but I should be able to assign a PR to someone in the assignee field, which is unrelated to whether `Issue` feature is enabled or disabled. 

I can assign the same PR in the website, so it should be a bug in the app, not the pull request.

Do you mean here is the incorrect place to submit a bug? I found the bug option in filling the form and thought here it the place to report the bug...

**Reply by guoxiao** [reply]
Submitted a ticket in https://support.github.com/contact/bug-report instead.

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @guoxiao ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by blackmonchasemining** [reply]
Hello

**Comment by Hoseinmj** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Select Feature Area
> 
> - [X] Issues
> - [X] Projects
> 
> ### Body
> 
> I want to assign a pull request in the Android app, but it says "Issues are disabled" and failed to assign. The assign features works well in the web so I think it's a bug?
> 
> ### Guidelines
> 
> - [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Canoonalmarridxb** [comment]
H![image](https://user-images.githubusercontent.com/93724787/236614596-cf976bb8-46a7-4a0c-b97d-8e92404f4701.png)
